packages/webapp/data/contract/requests/getMetadataAttributes.ts

```diff
@@ -0,0 +1,77 @@
+import { Address } from 'viem'
+import { readContracts } from 'wagmi/actions'
+
+import { metadataAbi } from '../../contract/abis'
+import { CHAIN_IDS } from '@constants/types'
+
+export const getMetadataAttributes = async (
+  metadata: Address,
+  finalTokenId: bigint,
+  chainId: CHAIN_IDS
+) => {
+  let lengthRequests = []
+  for (let currentTokenId = 0n; currentTokenId <= finalTokenId; currentTokenId++) {
+    // @ts-ignore
```

Comment:
what is being ignored here?

Reply:
This is how nouns.build wrote the code but the type of `lengthRequests` was being inferred as `never[]`. Changing it to `any[]` and remover the ts-gnore comment